nion/swift/ImageCanvasItem.py

```diff
@@ -639,6 +639,43 @@ async def _reactor_loop(self, r: Stream.ValueChangeStreamReactorInterface[MouseP
                 change_display_properties_task.commit()
 
 
+class ZoomMouseHandler(MouseHandler):
+    def __init__(self, image_canvas_item: ImageCanvasItem, event_loop: asyncio.AbstractEventLoop) -> None:
+        super().__init__(image_canvas_item, event_loop)
+
+    async def _reactor_loop(self, r: Stream.ValueChangeStreamReactorInterface[MousePositionAndModifiers],
+                            image_canvas_item: ImageCanvasItem) -> None:
+        delegate = image_canvas_item.delegate
+        assert delegate
+
+        # get the beginning mouse position
+        value_change = await r.next_value_change()
+        value_change_value = value_change.value
+        assert value_change.is_begin
+        assert value_change_value is not None
+        image_position: typing.Optional[Geometry.FloatPoint] = None
+        mouse_pos, modifiers = value_change_value
+
+        if modifiers.control:
```

Comment:
This should be `modifiers.alt`, mapping to the Alt key on Windows and the Option key on macOS. See https://helpx.adobe.com/illustrator/using/tool-techniques/zoom-tool.html.